SDK Showcase/SDK Showcase/Views/ContentView.swift

```diff
@@ -141,6 +142,17 @@ private extension ContentView {
 
 //MARK: - Private
 private extension ContentView {
+    var userStatus: String {
+        switch (system.registrationState, system.authenticationState) {
+        case (.registered, .authenticated(let userId)):
+            "👤 User authenticated as \(userId)"
+        case (.registered, _):
+            "👥 At least one user is registered"
+        default:
+            "🚫 No registered"
+        }
+    }
+    
```

Comment:
haha I know what you misunderstood now, last time I meant to remove the authentication state at all and combine it with registration state, as one of the options